changebot/blueprints/tests/test_stale_pull_requests.py

```diff
@@ -117,6 +117,22 @@ def test_close(self):
         self.submit_comment.assert_called_with(expected)
         assert self.close.call_count == 1
 
+    def test_close_noclose(self):
```

Comment:
@astrofrog , I wanted to a similar test for `test_pull_request_checker.py` for the change log config, but not sure how since there is no equivalent test class set up and I am unfamiliar with the `app` stuff to set one up myself.